ush/forecast_postdet.sh

```diff
@@ -461,14 +461,18 @@ WW3_postdet() {
   # Loop for gridded output (uses FHINC)
   local fhr fhr3 vdate FHINC ww3_grid
   fhr=${FHMIN_WAV}
-  fhinc=${FHOUT_WAV}
+  if (( FHMAX_HF_WAV > 0 && FHOUT_HF_WAV > 0 && fhr < FHMAX_HF_WAV )); then 
```

Comment:
Prefer `[[ ]]` logical tests as they offer better logging than `(( ))` tests.  (Existing logical tests can stay as they are for now and will be replaced at a later time.)

```suggestion
  if [[ ${FHMAX_HF_WAV} -gt 0 && ${FHOUT_HF_WAV} -gt 0 && ${fhr} -lt ${FHMAX_HF_WAV} ]]; then
```